src/Microsoft.AspNet.Authentication.OpenIdConnect/project.json

```diff
@@ -20,6 +20,7 @@
         },
         "dnxcore50": {
             "dependencies": {
+                "System.Dynamic.Runtime": "4.0.11-beta-*",
```

Comment:
Note: this will be reverted when @tushargupta51 updates the IdentityModel dependencies.